### PR TITLE
Fix e-mail list subscribe/unsubscribe and FAQ links

### PR DESCRIPTION
**Pull request by SataQiu**
Fix e-mail list subscribe/unsubscribe and FAQ links

Ref:
图片

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -131,15 +131,12 @@
...
 
- wuhan2020 邮件列表：[wuhan2020@googlegroups.com](https://groups.google.com/forum/#!forum/wuhan2020) ([订阅](mailto:dev-subscribe@dubbo.apache.org), [退订](mailto:dev-unsubscribe@dubbo.apache.org), [存档](https://groups.google.com/forum/#!forum/wuhan2020))
-
+wuhan2020 邮件列表：[wuhan2020@googlegroups.com](https://groups.google.com/forum/#!forum/wuhan2020) ([订阅](mailto:wuhan2020+subscribe@googlegroups.com), [退订](mailto:wuhan2020+unsubscribe@googlegroups.com), [存档](https://groups.google.com/forum/#!forum/wuhan2020))
 
 欢迎大家加入共同探讨各类技术或非技术类问题，让我们大家一起齐心协力，众志成城，共克时艰！
 
 # FAQ常见问题
 
-已经整理整个项目的FAQ，[请查看](./FAQ.md)
```
Comment (lovepoem) [inline README.md]: Now the new url is here :https://community.wuhan2020.org.cn/zh-zh/docs/overview/faq.html

File: README_EN.md [hunk 2]
```diff
@@ -109,14 +109,12 @@ Click here to join the [Slack Communication Group](https://join.slack.com/t/wuha
...
 # E-mail List
 
-wuhan2020 e-mail list: [wuhan2020@googlegroups.com](https://groups.google.com/forum/#!forum/wuhan2020) ([subscribe](mailto:dev-subscribe@dubbo.apache.org), [unsubscribe](mailto:dev-unsubscribe@dubbo.apache.org), [archive](https://groups.google.com/forum/#!forum/wuhan2020))
+wuhan2020 e-mail list: [wuhan2020@googlegroups.com](https://groups.google.com/forum/#!forum/wuhan2020) ([subscribe](mailto:wuhan2020+subscribe@googlegroups.com), [unsubscribe](mailto:wuhan2020+unsubscribe@googlegroups.com), [archive](https://groups.google.com/forum/#!forum/wuhan2020))
 
 Welcome to join together to explore all kinds of technical and non-technical issues, let's all work together and united to overcome the difficulties and fight against the virus!
 
 # FAQ
 
-Selected FAQ，[click here](./FAQ.md).
```
Comment (lovepoem) [inline README_EN.md]: Now the new url is here :https://community.wuhan2020.org.cn/en-us/docs/overview/faq.html

File: README_IT.md [hunk 3]
```diff
@@ -99,8 +99,6 @@ Invitiamo tutti a partecipare al gruppo su Slack per comunicare al meglio su que
 
 # FAQ (Sezione in fase di sviluppo, restate sintonizzati! Abbiamo bisogno di volontari che contribuiscano a questa sezione, chi è interessato è invitato a contattarci!)
 
-FAQ generiche sul progetto finora raccolte: [clicca qui](./FAQ.md).
-
```
Comment (lovepoem) [inline README_IT.md]: Now the new url is here :https://community.wuhan2020.org.cn/it-it/docs/overview/faq.html

File: README_JP.md [hunk 4]
```diff
@@ -114,8 +114,6 @@
 
 # FAQ （更新中，ボランティア募集中）
 
-よくある質問，[クリック](./FAQ.md)
```
Comment (lovepoem) [inline README_JP.md]: Now the new url is here :https://community.wuhan2020.org.cn/ja-jp/docs/overview/faq.html

File: README_TW.md [hunk 5]
```diff
@@ -131,14 +131,12 @@
...
 # 郵件列表
 
-wuhan2020 郵件列表: [wuhan2020@googlegroups.com](https://groups.google.com/forum/#!forum/wuhan2020) ([訂閱](mailto:dev-subscribe@dubbo.apache.org), [退訂](mailto:dev-unsubscribe@dubbo.apache.org), [存檔](https://groups.google.com/forum/#!forum/wuhan2020))
+wuhan2020 郵件列表: [wuhan2020@googlegroups.com](https://groups.google.com/forum/#!forum/wuhan2020) ([訂閱](mailto:wuhan2020+subscribe@googlegroups.com), [退訂](mailto:wuhan2020+unsubscribe@googlegroups.com), [存檔](https://groups.google.com/forum/#!forum/wuhan2020))
 
 歡迎大家加入共同探討各類技術或非技術類問題，讓我們大家一起齊心協力，眾志成城，共體時艱！
 
 # FAQ常見問題
 
-已經整理整個專案的FAQ，[請檢視](./FAQ.md)
```
Comment (lovepoem) [inline README_TW.md]: Now the new url is here :https://community.wuhan2020.org.cn/zh-cn/docs/overview/faq.html

**Comment by lovepoem** [comment]
这个文章是 README的copy 看起来也有这个问题，麻烦去fix下https://community.wuhan2020.org.cn/zh-cn/docs/dev/quickstart.html

**Comment by SataQiu** [comment]
> 这个文章是 README的copy 看起来也有这个问题，麻烦去fix下https://community.wuhan2020.org.cn/zh-cn/docs/dev/quickstart.html

OK! I'll have a look!

**Pull request merged**